bundles/binding/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/HueGenericBindingProvider.java

```diff
@@ -84,10 +84,12 @@ public void processBindingConfiguration(String context, Item item,
 
 		super.processBindingConfiguration(context, item, bindingConfig);
 
-		try {
-
-			if (bindingConfig != null) {
+    if( bindingConfig == null || bindingConfig.length() < 1 ) {
+        logger.warn("No settings configured for Hue binding for item '"+item+"'.");
+        return;
+    }
```

Comment:
Thanks for this, Chris.  Could you use the formatter in the Eclipse IDE (or equivalent), and also use parameterized logging instead of string concatenation, like
```java
logger.warn("No settings configured for Hue binding for item '{}'.", item);
```